### PR TITLE
add skip-if-changed to presubmit jobs

### PR DESCRIPTION
**Pull request by AsherShoshan**

### CODE REVIEW

**Comment by AsherShoshan** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-presubmit

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3 periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-private-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3 periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-private-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3 periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-private-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-private-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3 periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-private-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3 periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3 periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3 periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3 periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-private-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-private-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-critical-high-f3

File: ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ promotion:
 releases:
   latest:
     integration:
-      name: "4.13"
+      name: "4.14"
```
Comment (amalykhi) [inline ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:24]: lgtm

**Pull request merged**